### PR TITLE
doc: add minutes for meeting Sep 13 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-09-13.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+## Previous Meeting Review
+
+We skipped this as we had a packed agenda.
+
+## Minutes
+
+### nodejs/TSC
+* New TSC Director/Chair Elections [#326](https://github.com/nodejs/TSC/issues/326)
+ * No objections, Myles acclaimed as Directory, Michael acclaimed as Chair
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: Director

Comment (thefourtheye) [reply meetings/2017-09-13.md]: Would it be better to have it as Michael Dawson, as we had Michaël and Michael in the meeting today?

File: meetings/2017-09-13.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+* TSC Meeting Schedule [#336](https://github.com/nodejs/TSC/issues/336)
+ * We should revisit quarterly, ideally aligned with 
+   daylight savings times changes.
+ * Agreed we should move forward with the proposed times.
+
+* 2FA for the entire org [#301](https://github.com/nodejs/TSC/issues/301)
+  * We have consensus on step 1, will take of agenda.
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: I always get confused, is it `of` or `off`?

File: meetings/2017-09-13.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  * Myles, explore usb keys, could foundation fund ?
+  * Rich, is has not been an issue so far, can we defer until we see
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: it has not

File: meetings/2017-09-13.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    if it will be an issue.
+  * Matteo, biggest issue is telling and reminding people to do it.
+  * Matteo, would be good to have  policy that if you need financial
+    support you can request it.
+
+* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
+  * Michael gave overview and request to get involved in issue
+  * Mateo highlighted Metrics as one where we might do more.
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: Matteo

File: meetings/2017-09-13.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+* Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+  * prefer not to tack on properties to people’s object
+  * having bundled into separate objects is better
+  * anna, Brian are you ok with current state ?  
+  * Brian, _ not ideal but if we can’t agree on any alternatives yes
+  * Can’t explain how it works without including discussions of
+    internal state machine.
+  * If we have separate object that refers to existing one, but
+    Matteo believes this may be an issue due to additional allocation
+  * Jeremiah just consider _XX for writable and readable part of
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: Nit: I think the `_` has to be escaped here.

File: meetings/2017-09-13.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+  * Research this week on risk/reward and push decision to LTS/Release
+    working group for informed decision on upcoming Monday LTS/Release meeting
+
+### nodejs/security-wg
+
+* CVE management process for Node.js [#33](https://github.com/nodejs/security-wg/issues/33)
+  * Michael gave an overview of the proposal. Wanted to make sure TSC
+    was comfortable moving forward. 
+  * We agreed to leave on agenda, take discussion back to github
+    for 1 week and discuss in the meeting gnext week.
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: next week

File: meetings/2017-09-13.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+* Requesting Rod to step down or be removed from the CTC [#165](https://github.com/nodejs/CTC/issues/165)
+  * Ongoing discussion over last week
+  * Most appropriate to discuss next steps in private section of meeting
+
+## Q&A, Other
+
+* not seen simple user access control with react ?
+  * we are not sure what the questions is.
+  * try using the help nodejs/repo
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: Nit: nodejs/help repo

File: meetings/2017-09-13.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+  * Matteo will open an issue to collect votes. 
+    
+# deps: backport V8 update to 6.1 https://github.com/nodejs/node/pull/15393
+  * Land 6.1 into 8 LTS
+  * V8 team has worked on abi compatible patches (unclear if this
+    is complete yet)
+  * V8 reports that 6.1 has a larger number of fixes that would help
+    ongoing maintenance
+  * Rod, originally we said 2 months at one point
+  * We would need to set end Oct at LTS release date
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: Nit: as LTS release date?

File: meetings/2017-09-13.md (hunk 1 above)
Comment (Trott) [reply meetings/2017-09-13.md]: `acclaimed` is the wrong verb here. (It means praised.) I think `named` would be a better word choice. @mhdawson Would you prefer we push nits like that ourselves, or do you prefer to handle them so you can evaluate them?

File: meetings/2017-09-13.md (hunk 2 above)
Comment (Trott) [reply meetings/2017-09-13.md]: `off`

File: meetings/2017-09-13.md (hunk 1 above)
Comment (Trott) [reply meetings/2017-09-13.md]: Full names or else GitHub handles seem like the way to go to avoid ambiguity.

Comment (mhdawson) [reply meetings/2017-09-13.md]: I'm ok with people pushing the nits themselves.

File: meetings/2017-09-13.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: yup typo in this case.

File: meetings/2017-09-13.md (hunk 3 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: fixed

File: meetings/2017-09-13.md (hunk 4 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: Fixed

File: meetings/2017-09-13.md (hunk 5 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: I looked at in "view" and it seemed ok.

File: meetings/2017-09-13.md (hunk 6 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: fixed

File: meetings/2017-09-13.md (hunk 7 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: fixed

File: meetings/2017-09-13.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2017-09-13.md]: fixed

**Comment by mhdawson** [comment]
pushed update to address comments so far.

File: meetings/2017-09-13.md [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+
+## Previous Meeting Review
+
+We skipped this as we had a packed agenda.
+
+## Minutes
+
+### nodejs/TSC
+* New TSC Director/Chair Elections [#326](https://github.com/nodejs/TSC/issues/326)
+ * No objections, Myles Borrins named as Director, Michael Dawson named as Chair
```
Comment (thefourtheye) [inline meetings/2017-09-13.md]: Myles Borins

Comment (mhdawson) [reply meetings/2017-09-13.md]: Fixed

**Pull request merged**